src/audio/eq_iir/llext/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2024 Intel Corporation.
```

Comment:
I'm assuming this file will be shrunk/replaced when Zephyr llext cmake helpers are used ?